lnrpc/routerrpc/routing_config.go

```diff
@@ -47,4 +47,8 @@ type RoutingConfig struct {
 	// McFlushInterval defines the timer interval to use to flush mission
 	// control state to the DB.
 	McFlushInterval time.Duration `long:"mcflushinterval" description:"the timer interval to use to flush mission control state to the DB"`
+
+	// AlwaysIntercept determines whether the HTLC interceptor is registered
+	// regardless of whether the RPC is called or not.
+	AlwaysIntercept bool `long:"alwaysintercept" description:"Whether to always intercept HTLCs, even if no stream is attached"`
```

Comment:
If `AlwaysIntercept` is on, that also means that everything will remain on hold until the interceptor connects. If the interceptor process isn't running, that could mean a lot of held htlcs along the routes.

How about an intermediate solution? If there's no interceptor, packets will be forwarded as usual - unless that packet has been offered to the interceptor before (it is a replay). Replays will remain in the queue and must be re-offered to the interceptor.

Reply:
For combined routing/gateway nodes that intercept traffic to an internal destination, it would be nice if regular routing doesn't die completely when the interceptor has problems.

Reply:
I think the desired behaviour here is very use-case specific, which is partially why I hadn't sent this PR before 😅. For Muun, having HTLCs be handled without the interceptor is almost certainly a bad idea, and can cause noticeable downtime for users when the interceptor is being updated. 

IMO while this puts a stronger requirement on the interceptor process, it already has a quite high bar of implicit requirements and this actually makes them clearer. The interceptor has a high-uptime requirement for any non-trivial use case I can think of.

Reply:
Indeed, there are plenty of use cases for the interceptor with varying requirements. The high uptime requirement seems reasonable for now. Most important thing is that nothing can get lost.